README.md

```diff
@@ -1,3 +1,23 @@
 # c360-python-client
 
 A python client for c360 projects.
+
+## Configuration
+
+This library can be configured by setting environment variables.
+
+The following configuration is mandatory:
+
+- `C360_TENANT` - the name of the tenant
+
+The following configurations are optional:
+
+- `C360_STAGE` - the target stage of the project. Can either be "prod" or "staging. Defaults to "prod"
```

Comment:
typo here:
`Can either be "prod" or "staging". Default to "prod"`

btw, do we need `dev` ?

Reply:
We don't have a `dev` api online so we don't need to support `dev`.